static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -67,8 +74,9 @@ check out the following:`,
   },
   [ErrorHelpType.HANDLE_HARD_NAVIGATE_ERROR]: {
     resources: {
-      description: t(
-        'Handle hard navigation errors occur in Next.js applications when trying to redirect to the same page. To learn more about how to fix these errors, check out these resources:'
+      description: tct(
+        '[errorType] occur in Next.js applications when trying to redirect to the same page. To learn more about how to fix these errors, check out these resources:',
+        {errorType: <b>Handle hard navigation errors</b>}
```

Comment:
nit: `errorsType` or `errorTypes` to indicate it expect phrase that works as a plural?

Reply:
thanks! will change